src/frontend/package.json

```diff
@@ -10,20 +10,22 @@
     "dev": "node --max_old_space_size=8192 node_modules/.bin/next dev",
     "local-dev": "API_URL=http://backend.genepinet.localdev:3000 npm run dev --  -p 8000",
     "tests": "jest --config jest/config.js",
-    "e2e": "mkdir -p /tmp/screenshots; DEBUG=pw:api npx playwright test --output /tmp/screenshots -c tests/features/playwright.config.ts -j 2 tests/features",
+    "e2e": "mkdir -p /tmp/screenshots; DEBUG=pw:api npx playwright test --output /tmp/screenshots -c e2e/config/ci.config.ts",
     "e2e-staging-from-local": "AWS_PROFILE=genepi-dev FRONTEND_URL=https://staging.genepi.czi.technology DEPLOYMENT_STAGE=staging make e2e",
     "type-check": "tsc --noEmit",
     "lint": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx} --quiet --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
     "lint-ci": "concurrently \"node_modules/.bin/eslint **/*.{ts,tsx}\" \"node_modules/.bin/stylelint '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\""
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.18.9",
     "@emotion/css": "^11.1.3",
     "@emotion/react": "^11.1.5",
     "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/styles": "^4.11.3",
+    "@redux-devtools/extension": "^3.2.3",
```

Comment:
can this be a devdependency?

Reply:
I only updated redux-devtools-extensions to @redux-devtools/extension because it has been migrated and the former is no longer maintained but I can move to devDependency as well